internal/api/v2/FEATURE_ROLLOUT.md

```
@@ -0,0 +1,228 @@
+showenrichments: true       # Show season/expectedness badges and external links:
+                            #   - All About Birds (Cornell Lab)
+                            #   - Xeno-canto (recordings)
+showsimilarspecies: true    # Show similar species comparison modal with:
+                            #   - Scientific names
+                            #   - Side-by-side descriptions
+                            #   - Identification tips
+```
+
+**Note:** These are independent toggles. You can show notes but hide comparisons, for example.
+
+## Monitoring During Rollout
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)